gradle/release.gradle

```diff
@@ -9,12 +9,15 @@ In case the release process from Travis CI fails, please try:
     preventing version.properties to be bumped.
   3. delete top release notes paragraph(s) so that they are recreated
     - useful for recreating/merging release notes
+
+Testing release locally:
+  (this section of docs is in progress)
+  1. Configure env TRAVIS_BRANCH, GH_TOKEN, MOCKITO_BINTRAY_API_KEY
+  2. run './gradlew release -PdryRun'
 """
 
 def skipReleaseCommitMessage = '[ci skip-release]'
 
-def customReleaseVersionPattern = /.*\[ci-release ([^\]]+)\].*/
```

Comment:
I'm tweaking my tomorrow presentation about Continuous Delivery right now, so not today, but in general why not :).